packages/adapter-better-sqlite3/src/better-sqlite3.ts

```diff
@@ -200,8 +200,8 @@ export class PrismaBetterSQLite3Adapter extends BetterSQLite3Queryable<StdClient
 }
 
 type BetterSQLite3InputParams = BetterSQLite3Options & {
-  url: ':memory:' | (string & {})
-  shadowDatabaseURL?: ':memory:' | (string & {})
```

Comment:
Let's leave that in to make it clear we support those special in memory database syntax. ;)